src/components/get-footer.js

```diff
@@ -1,4 +1,8 @@
 // Create footer component
+
+// Config file
+import { config } from "../config.js";
```

Comment:
i think you can import a variable from config object, not the whole object(as you don't need it)